docs/kit/hardware/kit-definition.md

```diff
@@ -1,57 +0,0 @@
----
-original:
-  authors: Richard Barlow
-  url: https://bitbucket.org/richardbarlow/sr-kit-coord/wiki/Kit%20Definition
----
-# Kit Definition
-
-This page serves as the canonical list of parts constituting a Student Robotics kit.
-
-## Hardware
-
-Part                     | Inventory Asset Type  | Qty |
```

Comment:
We should probably keep this. This is the most complete list of what is in a kit (inc. things like the gauge and length).